packages/test-runner-commands/browser/commands.mjs

```diff
@@ -48,3 +48,7 @@ export function emulateMedia(media) {
 export function setUserAgent(options) {
   return executeServerCommand('set-user-agent', options);
 }
+
+export function sendKeys(options) {
```

Comment:
Is there a way to get a type for options here so our IDEs will tell us how to use the API? Including a link to the `type` vs `press` distinction in the Puppeteer API?

Reply:
There is a handwritten type definition file for this, to avoid having to run a second build for the browser code.

https://github.com/modernweb-dev/web/blob/master/packages/test-runner-commands/browser/commands.d.ts

Reply:
I'll update this. Hopefully it's easy enough to share the types between this and the actual plugin so that they don't have live in multiple places.

Reply:
@justinfagnani you wouldn't happen to have an example of a comment with links in it that I could replicate here? Not sure if you just want the raw URLs or if something else would be more useful here.

Reply:
Added expansive comments to the the `*.d.ts` file, but I'm not sure if that means it'll 100% pass into local files or not...thoughts?